ha/util/consul_kv_store.py

```diff
@@ -176,3 +178,8 @@ def delete(self, key: str = "", recurse: bool = False):
         data = self.get(key)
         self._consul.kv.delete(self._prepare_key(key), recurse=recurse)
         return data
+
+    def get_keys(self, prefix):
```

Comment:
Please add docstring for this function. Also, can you tell me why its needed?

Reply:
Done.

Reply:
Thanks!